tests/ocf-meta-inf_inc/EPUB/content_001.xhtml

```diff
@@ -0,0 +1,8 @@
+<html xmlns="http://www.w3.org/1999/xhtml" xmlns:epub="http://www.idpf.org/2007/ops" xml:lang="en">
```

Comment:
Excellent test, thank you! Can you please rename the files, though? An underscored section is only necessary if you need to *distinguish* one test from another, for the same test anchor (like if you have one MUST, but want to write two separate tests). This is 1 anchor and 1 test, so it should just be something like ocf-metainf-inc (in the filename, I propose not hyphenating metainf, because it's not hyphenated in https://www.w3.org/TR/epub-rs-33/#sec-container-metainf-inc, but I would be happy either way).

Reply:
oh, you meant,
1. rename test id to "ocf-metainf-inc"
2. rename test folder to "ocf-metainf-inc"
ok. lemme make the change.